rpcs3/rpcs3qt/main_window.ui

```diff
@@ -264,6 +264,7 @@
     <addaction name="separator"/>
     <addaction name="showHiddenEntriesAct"/>
     <addaction name="separator"/>
+    <addaction name="showCompatibilityInGridAct"/>
```

Comment:
You forgot to add a `<addaction name="separator"/>` after the showCompatibilityInGridAct